TombEngine/Game/control/box.cpp

```diff
@@ -820,45 +813,46 @@ void CreatureDie(short itemNumber, bool explode)
 
 int BadFloor(int x, int y, int z, int boxHeight, int nextHeight, short roomNumber, LOTInfo* LOT)
 {
-	FloorInfo* floor = GetFloor(x, y, z, &roomNumber);
+	auto floor = GetFloor(x, y, z, &roomNumber);
 	if (floor->Box == NO_BOX)
-		return true;
```

Comment:
Why return values are changed to integer? I think function declaration is wrong (should also be changed to bool).